tradeexecutor/analysis/trade_analyser.py

```diff
@@ -140,6 +147,25 @@ class TradePosition:
     #: but we cache it by hand to speed up processing
     closed_at: Optional[pd.Timestamp] = None
 
+    #: The total dollar value of the portfolio when the position
+    #: was opened
+    portfolio_value_at_open: Optional[USDollarAmount] = None
+
+    #: What is the maximum risk of this position.
+    #: Risk relative to the portfolio size.
+    loss_risk_at_open_pct: Optional[float] = None
```

Comment:
Should use dollar value here.

Percent is something we can calculate from dollars, but calculating dollars from percents do not make sense